flink-kubernetes-operator-autoscaler/src/main/java/org/apache/flink/kubernetes/operator/autoscaler/JobAutoScalerImpl.java

```diff
@@ -148,4 +162,22 @@ private AutoscalerFlinkMetrics getOrInitAutoscalerFlinkMetrics(
                         new AutoscalerFlinkMetrics(
                                 ctx.getResourceMetricGroup().addGroup("AutoScaler")));
     }
+
+    private void initRecommendedParallelism(
+            Map<JobVertexID, Map<ScalingMetric, EvaluatedScalingMetric>> evaluatedMetrics) {
+        evaluatedMetrics.forEach(
+                (jobVertexID, evaluatedScalingMetricMap) ->
+                        evaluatedScalingMetricMap.put(
+                                RECOMMENDED_PARALLELISM,
+                                evaluatedScalingMetricMap.get(PARALLELISM)));
+        LOG.debug("Recommended parallelisms where initialized  {}", evaluatedMetrics);
```

Comment:
I think the initialized / reset debug logs are an overkill. They don't provide any valuable info and are called in every iteration